scripts/schema-manager/manifest-loader.ts

```diff
@@ -0,0 +1,101 @@
+import { TigrisSchema } from '@tigrisdata/core/dist/types';
```

Comment:
this manifest loader looks complex to me. What about moving this functionality to our client? And let the user pass the model directory during client creation?

Reply:
Good idea

Reply:
Let's do it

Reply:
Yes, I was thinking of moving this as a plugin in SDK. Will primarily be used by Nextjs developers as it follows similar file-system based routing. This will still need a small setup script and hook into **dev** and **build** triggers.

Reply:
I think let's add it, would be easy to get feedback once we have this functionality. Otherwise, every dev needs to write this custom loader. This can be leveraged in the nuxt framework as well. A small script makes sense to create/setup these models.